docs/authentication/client-credentials.md

```diff
@@ -29,10 +29,16 @@ Replace the `{CLIENT_ID}` and `{CLIENT_SECRET}` values in the example below with
 POST /connect/token HTTP/1.1
 Host: login.elfsquad.io
 Content-Type: application/x-www-form-urlencoded
-
-client_id={CLIENT_ID}&client_secret={CLIENT_SECRET}&grant_type=client_credentials&scope=Elfskot.Api
 ```
 
+Body (x-www-form-urlencoded):  
```

Comment:
@eelkekramer Perhaps it would be better to move this table outside the code sample tabs?

Reply:
Ty for the feedback. It was incorporated in newest version.